grammars/angular/syntaxes/template.json

```
@@ -0,0 +1,195 @@
+{
+	"scopeName": "template.ng",
+	"injectionSelector": "L:text.html -comment",
+	"patterns": [
+		{
```

```suggestion
    {
      "include": "template.blocks.ng",
    }
		{
```
This will be needed if the template block syntax isn't injected into `source.ts`. I'm planning to do this [upstream as well](https://github.com/angular/vscode-ng-language-service/pull/1953)

---

Gotcha, thanks for the heads up.